components/molecules/ContributorHighlight/contributor-highlight-card.tsx

```diff
@@ -218,12 +221,28 @@ const ContributorHighlightCard = ({
                   <BsLink45Deg size={22} />
                   <span>Copy link</span>
                 </DropdownMenuItem>
-                <DropdownMenuItem className="hidden rounded-md">
-                  <div className="flex gap-2.5 py-1  items-center pl-3 pr-7">
-                    <FaUserPlus size={22} />
-                    <span>Follow user</span>
-                  </div>
-                </DropdownMenuItem>
+                {loggedInUser ? (
+                  <DropdownMenuItem
+                    className={`rounded-md ${loggedInUser.user_metadata.user_name === user && "hidden"}`}
+                  >
+                    <div onClick={isError ? follow : unFollow} className="flex gap-2.5 py-1  items-center pl-3 pr-7">
+                      <FaUserPlus size={22} />
+                      <span>
+                        {!isError ? "unfollow" : "follow"} {user}
```

Comment:
Capitalize actions. Do we need the username after?

```suggestion
                        {!isError ? "Unfollow" : "Follow"} {user}
```

Reply:
Yes we do as seen here
![image](https://user-images.githubusercontent.com/62995161/229853597-6a43660f-2025-4064-a216-745e3b611128.png)